orange-datamodel/src/test/resources/minimally.empty.orange.json

```diff
@@ -122,6 +121,7 @@
     "purpleClonalityPlot": "",
     "purpleCopyNumberPlot": "",
     "purpleVariantCopyNumberPlot": "",
-    "purplePurityRangePlot": ""
+    "purplePurityRangePlot": "",
+    "purpleKataegisPlot": "plot/tumor_sample.somatic.rainfall.png"
```

Comment:
Why do we need to provide an actual path here while the other plots can be empty?